src/modules/pokemons/evolutions/Restrictions.ts

```diff
@@ -89,21 +90,20 @@ export const weatherRestrict = <T extends EvoFn>(evo: T) => (
 );
 
 export const timeRestrict = <T extends EvoFn>(evo: T) => (
-    startHour: number,
-    endHour: number,
+    dayCycleParts: DayCyclePart[],
```

Comment:
This should be renamed to `dayCyclePartRestrict`

Reply:
This was talking about the `const timeRestrict`, since it is restricted by DayCycleParts now and the name should match

Reply:
that's what i thought at first,  but then i thought the line was meaningful ^^ not easy to place that in a review : )